src/services/jwt.service.js

```diff
@@ -0,0 +1,50 @@
+import jwt from 'jsonwebtoken';
+
+const generateAccessToken = (user) => {
+  return jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '10m',
+  });
+};
+
+const generateRefreshToken = (user) => {
+  return jwt.sign(user, process.env.JWT_REFRESH_KEY);
+};
+
+const validateAccessToken = (token) => {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (e) {
+    return null;
+  }
+};
+
+const validateRefreshToken = (refreshToken) => {
+  try {
+    return jwt.verify(refreshToken, process.env.JWT_REFRESH_KEY);
+  } catch (e) {
+    return null;
+  }
+};
+
+const generateResetPasswordToken = (user) => {
+  return jwt.sign(user, process.env.JWT_RESET, {
```

Comment:
Ensure that the 'process.env.JWT_RESET' environment variable is correctly set and secure, as it is used for signing the reset password token.